screens/friends/Friends.tsx

```diff
@@ -115,8 +115,8 @@ const FriendsProvider = ({ children }: FriendsProviderProps) => {
     await Promise.all([
       getFriendsData(token),
       getUsersData(token),
-      getReceivedRequestsData(id, token),
-      getSentRequestsData(id, token),
+      getReceivedRequestsData(token),
+      getSentRequestsData(token),
     ]).then(
```

Comment:
Unrelated to the changes but you don't need to `await` if you use `then`, so you can remove it (and then also remove `async`)

Reply:
thanks, done